lib/cn/csched_sp3.c

```diff
@@ -432,6 +432,7 @@ sp3_log_progress(struct cn_compaction_work *w, struct cn_merge_stats *ms, bool f
         "kblk_readwait_ops=%u kblk_readwait_ms=%u "
         "vblk_dbl_reads=%ld "
         "queue_us=%lu prep_us=%lu build_us=%lu commit_us=%lu",
+        // clang-format off
```

Comment:
Why not in front of log_info(), will probably save some heartache later.  Feels weird here..

Reply:
Not sure what heartache it saves.  Having it here ensures the leading params are formatted and lined up.

Reply:
The heartache when clang-format decides to effing reformat the code above.  Just because it doesn't do it now doesn't mean it wont do it tomorrow....   I wonder what it'd do if we put the off/on comments in the log_xxx() macro...

Reply:
It tested from V13 to V16 with a variety of formats and there were no changes between releases.

Reply:
It's ugly and inconsistent with your other changes.  Plus, maybe I change something in the above lines and clang-format wrecks the entire thing.  And what about V17, 18, 19, ...?  Who knows?  Not to mention, I hate one-offs...

Reply:
fixed in recent push